.devops/templates/runpublishvrscreenshot.yml

```diff
@@ -0,0 +1,47 @@
+parameters:
+  - name: fluentVersion
+    type: string
+    default: v8
+  - name: storyPackageName
```

Comment:
Can we rename this parameter to something like `vrTestPackageName` to reflect that it's actually pertaining to the individual VR test packages as opposed to a singular story?

Reply:
Fixed